docs/guides/react-server-cli.md

```diff
@@ -157,7 +157,8 @@ export default (server, reactServerMiddleware) => {
 	server.use(compression());
 	server.use(bodyParser.urlencoded({ extended: false }));
 	server.use(bodyParser.json());
-	server.use(session(options));
+	server.use(helmet());
```

Comment:
I read through the documentation to update it with `compileOnStartup` stuff and noticed that the custom express middleware example wasn't updated previously.  I can remove it if it's an issue.